enclave/src/polkadex_gateway.rs

```diff
@@ -238,8 +239,9 @@ pub fn cancel_order(
     } else {
         return Err(GatewayError::UnableToRemoveOrder);
     }
-    error!("Unable to load the cancel cache pointer");
-    return Err(GatewayError::UndefinedBehaviour);
+    // error!("Unable to load the cancel cache pointer"); //TODO: @gautham why we need this?
```

Comment:
This will be necessary in the process_cancel_order because we're working with non blocking calls (see issue #65 for the discussion)

Reply:
But leave it like that, this function will need to be refactored anyway for the openfinex client call.